### ISSUE TITLE
Foveal response dynamics for osBioPhys, see t_coneMosaicFoveal.m

### ISSUE THREAD

**Issue by jamesgolden1** [isetbio/isetbio#171]
We discussed the foveal cone dynamics that Fred and I implemented in osBioPhys. A few months ago Fred shared the plot below from his data, as well as the biophysical model parameters for implementing the response dynamics.

![foveal-cones](https://cloud.githubusercontent.com/assets/8620201/19367844/e475dc4e-9151-11e6-9c1d-47ac02a80d9e.png)

I added a tutorial called t_coneMosaicFoveal that generates a similar plot. It seems there could be a difference between the impulse response for the peripheral cone in isetbio and the plot from the physiological data, although there may just be a vertical shift of the peripheral response curve.

![foveal-cones-isetbio](https://cloud.githubusercontent.com/assets/8620201/19367796/8ab66994-9151-11e6-8588-3f5b0aa98a18.png)

Peripheral parameters:
        sigma: 22
          phi: 22
          eta: 2000
        gdark: 20.5000
            k: 0.0200
            h: 3
        cdark: 1
         beta: 9
     betaSlow: 0.4000
            n: 4
          kGc: 0.5000
    OpsinGain: 10

Foveal parameters:
        sigma: 10
          phi: 22
          eta: 700
        gdark: 20.5000
            k: 0.0200
            h: 3
        cdark: 1
         beta: 5
     betaSlow: 0.4000
            n: 4
          kGc: 0.5000
    OpsinGain: 12

**Comment by npcottaris** [comment]
So the foveal cones have longer latency, and slower dynamics, but also a biphasic response?
Is this due to feedback from horizontal cells?

N.



On Oct 13, 2016, at 5:32 PM, jamesgolden1 <notifications@github.com<mailto:notifications@github.com>> wrote:


We discussed the foveal cone dynamics that Fred and I implemented in osBioPhys. A few months ago Fred shared the plot below from his data, as well as the biophysical model parameters for implementing the response dynamics.

[impulseresponsefovea]<https://cloud.githubusercontent.com/assets/8620201/19367646/e24b5a08-9150-11e6-9e5f-7078c58e6885.png>

I added a tutorial called t_coneMosaicFoveal that generates a similar plot. It seems there could be a difference between the impulse response for the peripheral cone in isetbio and the plot from the physiological data, although there may just be a vertical shift of the peripheral response curve.

[foveal-cones-isetbio]<https://cloud.githubusercontent.com/assets/8620201/19367796/8ab66994-9151-11e6-8588-3f5b0aa98a18.png>

Peripheral parameters:
sigma: 22
phi: 22
eta: 2000
gdark: 20.5000
k: 0.0200
h: 3
cdark: 1
beta: 9
betaSlow: 0.4000
n: 4
kGc: 0.5000
OpsinGain: 10

Foveal parameters:
sigma: 10
phi: 22
eta: 700
gdark: 20.5000
k: 0.0200
h: 3
cdark: 1
beta: 5
betaSlow: 0.4000
n: 4
kGc: 0.5000
OpsinGain: 12
